tests/vm_test.go

```diff
@@ -1778,20 +1795,3 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 	})
 })
-
-func getExpectedPodName(vm *v1.VirtualMachine) string {
-	maxNameLength := 63
-	podNamePrefix := "virt-launcher-"
-	podGeneratedSuffixLen := 5
-	charCountFromName := maxNameLength - len(podNamePrefix) - podGeneratedSuffixLen
-	expectedPodName := fmt.Sprintf(fmt.Sprintf("virt-launcher-%%.%ds", charCountFromName), vm.GetName())
-	return expectedPodName
-}
-
-func waitForResourceDeletion(k8sClient string, resourceType string, resourceName string) {
```

Comment:
I would vote against this commit. passing `k8sClient` as an arg is cumbersome, but it's generally better to make the dependency on an almost-global variable explicit. It leaves less room for surprises if somewhere else in the code changes it.